.github/workflows/premerge.yml

```diff
@@ -2,14 +2,14 @@ name: pre-merge
 
 on:
   # quick tests for pull requests and the releasing branches
```

Comment:
Does this mean runtest.sh / run_example_tests.sh / run_app_tests.sh are all quick tests?

Reply:
No, the change means that we'll need to issue a "/build" comment to run ci now